content/Contributing/Applications/AppArticleTemplate.md

```diff
@@ -15,11 +15,11 @@ Check back for updates!
 
 ## How to Use this Template
 
-Click **Edit Page** above or locate the Markdown file at [/Contributing/Applications/AppArticleTemplate.md](https://github.com/truenas/documentation/blob/master/Contributing/Applications/AppArticleTemplate.md) on the documentation [Github Repository](https://github.com/truenas/documentation).
+Click **Edit Page** above or locate the Markdown template at [`content/resources/AppArticleTemplate.md`](https://github.com/truenas/apps-web/blob/main/content/resources/AppArticleTemplate.md) in the [TrueNAS Apps Market Repository](https://github.com/truenas/apps-web).
```

Comment:
Do we want to review and alter the template to match how resource articles are currently configured on the apps-web repo?
```suggestion
Click **Edit Page** above or locate the Markdown template at [`content/resources/AppArticleTemplate.md`](https://github.com/truenas/apps-web/blob/main/content/resources/AppArticleTemplate.md) in the [TrueNAS Apps Market Repository](https://github.com/truenas/apps-web).
```